### PR TITLE
LPD-42921 Add new languages

### PR DESCRIPTION
**Pull request by thiago-buarqque**
https://liferay.atlassian.net/browse/LPD-42921

# Context

Our customer Sony asked to add these language ids

### CODE REVIEW

**Comment by pat270** [comment]
@thiago-buarqque LGTM! I forgot are we supposed to build lang? I remember Bchan mentioned we don't need to do that anymore in a thread some where.

**Comment by thiago-buarqque** [comment]
@pat270 just removed it. Better him to do it, it's 130k lines

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -10058,6 +10063,7 @@ language.ko=Korean
 language.lo=Lao
 language.lt=Lithuanian
 language.lv=Latvian
+language.mk_MK=Macedonian (Macedonia)
```
Comment (ethib137) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:10066]: Should we be adding a 4 letter code before we add the 2 letter base code? It looks like we only add 4 letter codes once there are multiple locales for the same language? Is that right?

Comment (thiago-buarqque) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:10066]: Ops, you are right. Good catch

Comment (thiago-buarqque) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:10066]: I did it, but for some reason things are not working properly. I am checking it with Brian Wulbern, he has some experience with this

Comment (brianwulbern) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:10066]: Yeah, following our other file names, we only need the `YY` in the full `xx_YY` format to make a distinction between dialects. See https://github.com/liferay/liferay-portal/tree/master/modules/apps/portal-language/portal-language-lang/src/main/resources/content

As for issues with how Macedonian is displaying, I'm discussing it in DM with Thiago. But from the contents of this pull it looks fine.

**Comment by thiago-buarqque** [comment]
@ethib137 After some issues generating the new properties file, we talked with Drew Brokke and he said Brian will regen everything later, because he needs to update a dependency first. 

I have implemented the suggested changes. Thanks @brianwulbern for your time

**Comment by thiago-buarqque** [comment]
ci:test:sf

**Comment by thiago-buarqque** [comment]
The failing line is there since 2015. Not sure if I should fix it

**Comment by ethib137** [comment]
Hey @thiago-buarqque , The SF error can be fixed like this: https://github.com/brianchandotcom/liferay-portal/commit/10400b13e1e8724cbd691984e6ff1b7258fa0926#diff-9a59ae6a78375a8681f5ef1282452ffd578a3518da66b76fd6ec85839e9ba968

You just need to update the line number to match the new line number after your changes.

**Comment by thiago-buarqque** [comment]
ci:test:sf

**Comment by thiago-buarqque** [comment]
ci:test:sf

**Comment by thiago-buarqque** [comment]
ci:test:relevant

**Comment by thiago-buarqque** [comment]
ci:test:sf

**Comment by thiago-buarqque** [comment]
ci:test:relevant

**Comment by brianwulbern** [comment]
Based on how we've done this in the past, the language add changes in this pull LGTM.

Most new languages added to liferay-portal in a long time!

**Review by ethib137 — APPROVED** [review]
LGTM. Thanks @thiago-buarqque !

**Comment by thiago-buarqque** [comment]
ci:forward:force

**Pull request closed without merging**